content/blog/deploying-super-mario-game-on-kubernetes-in-hpe-greenlake-for-private-cloud-enterprise.md

```diff
@@ -0,0 +1,477 @@
+│   └── service.yaml
+└── tetris
+    ├── deployment.yaml
+    └── service.yaml
+```
+
+T﻿ype the following commands to deploy *Super Mario* and *Tetris* to the namespace *cfe-games* in the cluster:
+
+```shell
+$ kubectl create ns cfe-games
+namespace/cfe-games created
+
+$ kubectl apply -f super-mario/ -n cfe-games
+deployment.apps/mario-deployment created
+service/mario-service created
+
+$ kubectl apply -f tetris/ -n cfe-games
+deployment.apps/tetris-deployment created
+service/tetris-service created
+```
+
+Type the command shown below to check the details of the game deployment:  
+
+```shell
+$ kubectl get all -n cfe-games
+NAME                                     READY   STATUS    RESTARTS   AGE
+pod/mario-deployment-96f79d8f-dw9hh      1/1     Running   0          19s
+pod/mario-deployment-96f79d8f-wsf7s      1/1     Running   0          13s
+pod/tetris-deployment-86d744fb47-7kmwl   1/1     Running   0          7s
+pod/tetris-deployment-86d744fb47-hqmgd   1/1     Running   0          10s
+
+NAME                     TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)   AGE
+service/mario-service    ClusterIP   10.104.144.88   <none>        80/TCP    22s
+service/tetris-service   ClusterIP   10.111.218.14   <none>        80/TCP    10s
+
+NAME                                READY   UP-TO-DATE   AVAILABLE   AGE
+deployment.apps/mario-deployment    2/2     2            2           24s
+deployment.apps/tetris-deployment   2/2     2            2           12s
+
+NAME                                           DESIRED   CURRENT   READY   AGE
+replicaset.apps/mario-deployment-96f79d8f      2         2         2       24s
+replicaset.apps/tetris-deployment-86d744fb47   2         2         2       12s
+```
+
+Two games, *mario-deployment* and *tetris-deployment*, are deployed in the cluster, each running with 2 Pod replicas by default. T﻿hey are exposed as the *ClusterIP* type o﻿f services, providing internal connectivity and solely being accessible from within the cluster.
+
+You can configure the [*Horizontal Pod Autoscaling* (HPA)](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/) in the cluster by using the K8s *HorizontalPodAutoscaler* resource. It will automatically scale the workload by deploying more Pods in the cluster according to game application memory or CPU usage.
+
+T﻿ype the following commend to check that all the game service endpoints have been populated:
+
+```shell
+$ kubectl get endpoints -n cfe-games
+NAME             ENDPOINTS                            AGE
+mario-service    10.192.3.118:80,10.192.4.32:80       60s
+tetris-service   10.192.3.119:3000,10.192.4.33:3000   50s
+```
+
+### Set up the load balancer with *MetalLB*
+
+You can install *MetalLB* and set up the load balancer in the K8s cluster by following up the blog post [Setting up the load balancer with MetalLB](https://developer.hpe.com/blog/set-up-load-balancer-with-metallb-in-hpe-greenlake-for-private-cloud-enterprise/).
```

Comment:
Similar to my comments in the previous blog post, change this to read:
"... in the K8s cluster by following the instructions shown in the blog post..."